livingpark_utils/livingpark_utils.py

```diff
@@ -745,18 +747,20 @@ def export_spm_segmentations(
                 os.path.join(self.data_cache_path, "inputs"),
                 os.path.join("outputs", "pre_processing"),
             )
-            output_file_c1 = output_file_name.replace("PPMI", "smwc1PPMI")
-            output_file_c2 = output_file_name.replace("PPMI", "smwc2PPMI")
+            output_file_c1 = output_file_name.replace("PPMI", "c1PPMI", 1)
+            output_file_c2 = output_file_name.replace("PPMI", "c2PPMI", 1)
 
             fig = plt.figure()
             display = nplt.plot_anat(
-                cut_coords=list(cut_coords), figure=fig, title=f"#{i}/{len(cohort)}"
-            )
-            display.add_overlay(output_file_c1, cmap="Reds", threshold=0.1, alpha=alpha)
-            display.add_overlay(
-                output_file_c2, cmap="Blues", threshold=0.1, alpha=alpha
+                anat_img=input_file, cut_coords=list(cut_coords) if cut_coords else None, figure=fig, title=f"#{i}/{len(cohort)}"
             )
 
+            if mni_space or show_gm:
+                display.add_overlay(output_file_c1, cmap="Reds", threshold=0.1, alpha=alpha)
+            
+            if mni_space or show_wm:
+                display.add_overlay(output_file_c2, cmap="Blues", threshold=0.1, alpha=alpha)
+
```

Comment:
The logic is not right: when showing in MNI space then anat_image is not shown (anat_image is in native space). Or else, use normalized anat_image. In addition, show_gm and show_wm should work wheter in mni_space or in native space.